core-bundle/src/Image/Studio/ImageResult.php

```diff
@@ -204,10 +204,10 @@ public function createIfDeferred(): void
             return;
         }
 
-        $resizer = $this->locator->get('contao.image.resizer');
+        $resizer = $this->locator->get('contao.image.legacy_resizer');
```

Comment:
@ausi: I am not sure about this change. The class is called `LegacyResizer` but there seems to be no non-legacy resizer?

Reply:
Not sure. This class will probably go away in Contao 5.0 and there will be no class at all anymore or it would be called resizer. So keeping the service called `contao.image.resizer` would be better here I think.